addons/web_editor/static/src/js/wysiwyg/wysiwyg_iframe.js

```diff
@@ -78,7 +78,11 @@ patch(Wysiwyg.prototype, {
         var self = this;
         this.$editable = $('<div class="note-editable oe_structure odoo-editor-editable"></div>');
         this.$el.removeClass('note-editable oe_structure odoo-editor-editable');
-        this.$iframe = $('<iframe class="wysiwyg_iframe o_iframe">').css({
+        this.$iframe = $(
+            `<iframe class="wysiwyg_iframe o_iframe ${
+                this.snippetsMenu && !this.snippetsMenu.folded ? "has_snippets_sidebar" : ""
```

Comment:
Note that we could toggle the class programatically here below to simplify init, not a big deal though.